src/platform/forms/save-in-progress/SaveInProgressIntro.jsx

```diff
@@ -28,8 +28,39 @@ import {
 } from '../../forms-system/src/js/constants';
 
 class SaveInProgressIntro extends React.Component {
+  getFormControls = savedForm => {
+    const { profile } = this.props.user;
+    const startPage = this.getStartPage();
+    const prefillAvailable = !!(
+      profile && profile.prefillsAvailable.includes(this.props.formId)
+    );
+    const isExpired = savedForm
+      ? moment.unix(savedForm.metadata.expiresAt).isBefore()
+      : true;
```

Comment:
Should `isExpired` be `false` if `savedForm` is falsy? In the original code `isExpired` is only assigned a value if `savedForm` exists, so I would expect it to be falsy if `savedForm` doesn't exist.

Reply:
I spent too much time over-thinking this... I'll change it to your suggestion. Thanks!